README.md

```diff
@@ -58,6 +58,8 @@ or
 | spanRejectionFilter | ((SpanData) -> SpanData?)? | Closure to modify or reject/ignore spans.  See example below.  | `nil` |
 | showVCInstrumentation | Bool | Enable span creation for ViewController Show events (not applicable to all UI frameworks/apps) | true |
 | screenNameSpans | Bool | Enable span creation for changes to the screen name | true |
+| slowFrameThreshold | CFTimeInterval | The SlowFrame Threshold is an optional configuration | 0.0169 |
+| frozenFrameThreshold | CFTimeInterval | The frozenFrame Threshold is an optional configuration | 0.7 |
```

Comment:
These descriptions are not useful, @Piyush23786. What do these threshold control?

+CC @seemk

Reply:
The SlowFrame Threshold is an optional configuration that marks all the frames that took more than the specified time as slow frames. The user needs to provide this value in milliseconds.

Reply:
That's great, @Piyush23786 . Could you please edit the descriptions in the table to match that?

Reply:
Done

Reply:
I still see the file as before.

Reply:
https://github.com/signalfx/splunk-otel-ios/pull/58/files